pallets/compounding-interest/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "compounding-interest"
+version = "0.1.0"
+authors = ["Joshy Orndorff"]
+edition = "2018"
+
+[features]
+default = ['std']
+std = [
+    'parity-scale-codec/std',
+    'frame-support/std',
+    'frame-system/std',
+    'sp-arithmetic/std',
+    'sp-runtime/std',
+    'sp-std/std',
+]
+
+[dependencies.substrate-fixed]
+git = 'https://github.com/encointer/substrate-fixed.git'
+version = "0.5.4"
```

Comment:
I think you'll want to refer to a release tag, not the version, because that's the upstream crate's version. I guess I'll keep that in case of upstream rebase.

I'd suggest
```
tag = "v0.5.4+sub_v0.1"
```